linux/aliases.bsh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env false bash
+
+if [[ $- != *i* ]]; then
+  source_once &> /dev/null && return 0
+fi
+
+#*# linux/aliases
+
+#**
+# ==================
+# Executable Aliases
+# ==================
+#
+# .. default-domain:: bash
+#
+# .. file:: aliases.bsh
+#
+# Since bash script do not use ``alias`` by default, common external programs are soft coded using an all caps variable name (e.g. ``tar`` is called using ``"${TAR}"``. Simply source :file:`aliases.bsh` and use the variables.
+#
+# Variables should default to the common executable name, and allow the user to override them.
+#
+# For example, :var:`SSH` defaults to ``ssh``. There are a number of ways to override this:
+#
+# * ``SSH=ssh2`` - Finds the command on the path called ``ssh2``
+# * ``SSH=/usr/local/bin/ssh`` - Uses a full path
+# * ``function ssh3(){ SSH_FLAG=3 ssh -v ${@+"${@}"}; }; export -f ssh3; SSH=ssh3`` - Use an exported function
+# * ``echo '#!/usr/bin/env bash' > ~/foo; echo 'ssh -v ${@}' >> ~/foo; SSH=~/foo``
+#
+# .. note::
+#
+#    ``SSH="ssh -v"`` will look for an executable called ``ssh -v``, not run ``ssh`` with the ``-v`` flag
+#
+# .. var:: GIT
+#
+# Name/path of git executable
+#
+# Instead of hard-coding "git" everywhere, use the variable :var:`GIT` so that when the need comes, it is easier to switch which executable gets called. Default is ``git``. Set to empty string to have any calls to git skipped.
+#
+# .. note::
+#
+#   When using the GIT env variable in this file, do not surround it with quotes as you normally would. Although this prevents using a path to ``git`` with a space in it, in exchange, it may be possible (eventually) to enable a dry-run-ish capability if GIT is set to ``echo git``. However, this feature does not currently work and even in the future may not work very well because this script is constantly cloning and cd'ing to new repos.
```

Comment:
This is no longer the recommendation, right?

Reply:
Correct